src/app/const/index.ts

```diff
@@ -1,3 +1,5 @@
+export const ANCHORS = { ABOUT_SCHOOL: 'about-school' };
```

Comment:
Why did you create a constant and ain't use it?

Reply:
Fixed